vng_api_common/exception_handling.py

```diff
@@ -102,7 +104,15 @@ def as_serializer(
         return serializer_class(instance=self)
 
     def log(self):
-        logger.exception("Exception %s ocurred", self._exc_id)
+        if self.logger:
+            self.logger.exception(
+                "Exception %s ocurred",
```

Comment:
Let's use an event name here instead
```suggestion
                "api_exception_handled",
```